src/qrq.c

```diff
@@ -1064,7 +1076,7 @@ static int calc_score (char * realcall, char * input, int spd, char * output, in
 			}
 		}
 		output[i]='\0';
-		if ((speed > 29) && !fixspeed) {speed -= 10;}
+		if ((speed > 29) && !fixspeed) {speed -= speedstep;}
```

Comment:
This check should be updated in order to avoid getting into negative speeds (or in general speeds below 30 lpm).

Reply:
Thanks for the review!

From reading the code, the intended minimum seems to be 20 rather than 30, and the README file confirms that. Updated in the just pushed version. (And also placed the edit-call dialog in the correct line now in F10-c mode.)